ldap/servers/slapd/log.c

```diff
@@ -441,6 +441,23 @@ log_set_logging(const char *attrname, char *value, int logtype, char *errorbuf,
     return LDAP_SUCCESS;
 }
 
+void
+log_external_libs_debug_print(char *buffer)
+{
+    slapi_log_error(SLAPI_LOG_FATAL, "OpenLDAP Client", "%s", buffer);
+}
+
+int
+log_external_libs_debug_set_log_fn(void)
+{
+    int rc = ber_set_option(NULL, LBER_OPT_LOG_PRINT_FN, log_external_libs_debug_print);
```

Comment:
How do you s-et the print fn for ldap? or does it just use stderr?

Reply:
It uses the LBER_OPT_LOG_PRINT_FN so it is set only one time (if I understand it correctly)
https://git.openldap.org/openldap/openldap/-/blob/master/libraries/libldap/print.c#L60